### PR TITLE
Multi env

### PR DESCRIPTION
**Pull request by pallenSL**
Fully parameterized.

### CODE REVIEW

File: dataCenter/ansible/env_local/group_vars/nodes.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+dynamodb_str: "SERVER=talksdev.ch20oh3elz3t.eu-west-1.rds.amazonaws.com;Database=sessions;UID=sessions;Password=;"
```
Comment (jebbinSCL) [inline dataCenter/ansible/env_local/group_vars/nodes.yml]: This isn't the dynamo db string, Its the mysql string?

File: dataCenter/ansible/roles/srm-gateway/tasks/main.yml [hunk 2]
```diff
@@ -14,4 +14,4 @@
     ports:
      - "8080:8080"
     volumes:
-     - /home/vagrant/ApiGateway.config:/service/ApiGateway.exe.config
+     - /home/vagrant/Gateway.config:/service/ApiGateway.exe.config
```
Comment (jebbinSCL) [inline dataCenter/ansible/roles/srm-gateway/tasks/main.yml]: Missed ApiGateway here?

File: dataCenter/ansible/roles/srm-sessions/templates/Sessions.config.j2 [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <configuration>
   <connectionStrings>
     <!-- Security provided by AWS security groups -->
-    <add name="DefaultConnection" connectionString="SERVER=talksdev.ch20oh3elz3t.eu-west-1.rds.amazonaws.com;Database=sessions;UID=sessions;Password=;" providerName="System.Data.SqlClient"/>
+    <add name="DefaultConnection" connectionString="{{ dynamodb_str }}" providerName="System.Data.SqlClient"/>
```
Comment (jebbinSCL) [inline dataCenter/ansible/roles/srm-sessions/templates/Sessions.config.j2]: Sessions uses mysql, not dynamodb

File: dataCenter/ansible/env_local/group_vars/nodes.yml (hunk 1 above)
Comment (pallenSL) [inline dataCenter/ansible/env_local/group_vars/nodes.yml]: D'Oh!  Thanks.

File: dataCenter/ansible/roles/srm-gateway/tasks/main.yml (hunk 2 above)
Comment (pallenSL) [inline dataCenter/ansible/roles/srm-gateway/tasks/main.yml]: It has to be ApiGateway.exe.config as long as that is the executable name.

File: dataCenter/ansible/roles/srm-sessions/templates/Sessions.config.j2 (hunk 3 above)
Comment (pallenSL) [inline dataCenter/ansible/roles/srm-sessions/templates/Sessions.config.j2]: Yeah, brainfart.  Changing.

**Comment by jebbinSCL** [comment]
:ship:

**Pull request merged**